_api-reference/validate.md

```diff
@@ -0,0 +1,205 @@
+`q` | String | Query in the Lucene query string syntax.
+
+## Example request
+
+The following example uses an index named `Hamlet`, created using the following `bulk` request:
+
+```
+PUT hamlet/_bulk?refresh
+{"index":{"_id":1}}
+{"user" : { "id": "hamlet" }, "@timestamp" : "2099-11-15T14:12:12", "message" : "To Search or Not To Search"}
+{"index":{"_id":2}}
+{"user" : { "id": "hamlet" }, "@timestamp" : "2099-11-15T14:12:13", "message" : "My dad says that I'm such a ham."}
+```
+{% include copy.html %}
+
+You can then use the Validate Query API to validate a query made against the index, as shown in the following example:
+
+```
+GET hamlet/_validate/query?q=user.id:hamlet
+```
+{% include copy.html %}
+
+The query can also be sent as a request body, as shown in the following example:
+
+```
+GET hamlet/_validate/query
+{
+  "query" : {
+    "bool" : {
+      "must" : {
+        "query_string" : {
+          "query" : "*:*"
+        }
+      },
+      "filter" : {
+        "term" : { "user.id" : "hamlet" }
+      }
+    }
+  }
+}
+```
+{% include copy.html %}
+
+
+## Example responses
+
+Depending on the validity of the request query, OpenSearch responds that the query is true, as shown in the following example response where the `valid` parameter is shown as `true`:
+
+```
+{
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "failed": 0
+  },
+  "valid": true
+}
+```
+
+If the query does not pass validation, OpenSearch responds that the query is false. The following example uses a request query that includes a dynamic mapping not configured in the `hamlet` index:
+
+```
+GET hamlet/_validate/query
+{
+  "query": {
+    "query_string": {
+      "query": "@timestamp:foo",
+      "lenient": false
+    }
+  }
+}
+```
+
+OpenSearch responds with the following, where the `valid` parameter is shown as `false`:
+
+```
+{
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "failed": 0
+  },
+  "valid": false
+}
+```
+
+Certain query parameters can also affect what OpenSearch shows in the response. The following examples show how the [Explain](#explain), [Rewrite](#rewrite), and [all_shards](#rewrite-and-all_shards) query options affect the response.
+
+### Explain 
+
+The `explain` option returns information about why the query inside the explanations response option failed, as shown in the following response example:
+
+```
+{
+  "valid" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "failed" : 0
+  },
+  "explanations" : [ {
+    "index" : "_shakespeare",
+    "valid" : false,
+    "error" : "shakespeare/IAEc2nIXSSunQA_suI0MLw] QueryShardException[failed to create query:...failed to parse date field [foo]"
+  } ]
+}
+```
+
+
+### Rewrite
+
+When the query is valid, the explanation response option shows the string representation of the query. When `rewrite:` is set to `true`:, the explanation is more detailed showing the actual Lucene query that is executed.
```

Comment:
Same here. This needs a bit of rewriting, and I suspect that certain terms should be in code font.